docs/framework/additional-apis/toc.md

```diff
@@ -1,4 +1,12 @@
 # [Additional Class Libraries and APIs](index.md)
-## [s_isDebuggerCheckDisabledForTestPurposes Field](s-isdebuggercheckdisabledfortestpurposes-field.md)
-## [DataMemberFieldEditor Class](datamemberfieldeditor-class.md)
-## [DataMemberListEditor Class](datamemberlisteditor-class.md)
+## [System.Net.Connection Class](../../../docs/framework/additional-apis/connection.md)
```

Comment:
I'd prefer the TOC node that just opens and close since the full name in the TOC doesn't render too well